src/packages/frontend/jupyter/output-messages/mime-types/register.ts

```diff
@@ -65,7 +65,8 @@ export function getHandler(type: string): Handler {
   if (h != null) return h.handler;
   for (const typeRegexp in HANDLERS) {
     if (type.match("^" + typeRegexp + "$")) {
-      return HANDLERS[typeRegexp].handler;
+      const hRegex = HANDLERS[typeRegexp];
```

Comment:
This is a long way of writing `return HANDLERS[typeRegexp]?.handler;`

Reply:
well, that wouldn't return the `FallbackHandler` or do I miss something? I could rewrite all of that as well, but I didn't want to spend more time.